src/devices/apator_metra_erm30.c

```diff
@@ -0,0 +1,267 @@
+/** @file
+    Apator Metra E-RM 30 Electronic Radio Module for Residential Water Meters
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+
+/**
```

Comment:
change to
`/** @fn int apator_metra_erm30_decode(r_device *decoder, bitbuffer_t *bitbuffer)`